src/ntttcp.c

```diff
@@ -58,6 +58,7 @@ void default_ntttcp_test(struct ntttcp_test *test)
 	test->json_log_filename		= DEFAULT_JSON_LOG_FILE_NAME; /* "ntttcp-for-linux-log.json" */
 	test->quiet			= false;
 	test->verbose			= false;
+	test->tcp_nodelay       	= false;
```

Comment:
Can you pls move this parameter right after "fq_rate_limit"? Grouping attributes/settings together will help the code readability and maintainability.